### ISSUE TITLE
FP: Child method is exactly the same when USE is used

### ISSUE THREAD

**Issue by CZechBoY** [kalessil/phpinspectionsea#450]
Hi,
I have this code in parent:
```php
public function addCheckbox($name, $caption = null)
    {
        return $this[$name] = new Controls\Checkbox($caption);
    }
```
and this is in its child
```php
public function addCheckbox($name, $caption = null)
    {
        return $this[$name] = new Controls\Checkbox($caption);
    }
```
yes, the same. But in child is additionaly use statement
```php
use App\MyControls as Controls;
```
so I create different class than the parent.

**Comment by CZechBoY** [comment]
Also another example: I and parent are using private property that is set in other method than constructor.

**Comment by funivan** [comment]
Thanks for the report @CZechBoY 
Can not reproduce your bug. 
Can you try this code?
```php
  namespace {
    class A {
      public function test() {
        echo (new stdClass())->df;
      }
    }
  }
  namespace DF {
    use A as stdClass;
    class B extends \A {
      public function test() {
        echo (new stdClass())->df;
      }
    }
  }
```
Can you create short peace of code that has false-positive ;)

**Comment by CZechBoY** [comment]
OK, I'm unable to reproduce it on sandbox too.
The class with overriden method is in vendor, and 3 parents of it are in vendor too, then my 2 classes extends that top vendor class.

**Comment by kalessil** [comment]
My bad, I constantly forget to take into account imports and aliasing.

**Comment by kalessil** [comment]
Great news, this has been fixed previously in #104
